packages/plugins/prometheus/src/utils.ts

```diff
@@ -104,23 +115,34 @@ export function registerHistogram<LabelNames extends string>(
 }
 
 export function createHistogram<
+  Phases extends string[],
   LabelNames extends string,
   Params extends Record<string, any> = FillLabelsFnParams,
 >(options: {
   registry: Registry;
   histogram: Omit<HistogramConfiguration<LabelNames>, 'registers'>;
   fillLabelsFn: FillLabelsFn<LabelNames, Params>;
-}): HistogramAndLabels<LabelNames, Params> {
+  phases: Phases;
+  shouldObserve?: ShouldObservePredicate<Params>;
```

Comment:
Can you ensure that if I hover this a helpful `jsdoc` will appear to explains what this option is doing, and what the default is? 

Such as
```
  /**
   * A function that determines whether the metric should be recorded or not.
   * 
   * Defaults to `() => true`.
   */
```

Might need to be repeated 2 other places below.